spa/src/components/content/pages/Pages.test.tsx

```diff
@@ -28,14 +29,21 @@ jest.mock('hooks/useUser', () => ({
   default: jest.fn()
 }));
 
+jest.mock('hooks/useRequest', () => ({
+  __esModule: true,
+  default: jest.fn()
+}));
```

Comment:
```suggestion
jest.mock('hooks/useRequest');
```

This can also be done for the useUser mock too, it looks like.